src/main.rs

```diff
@@ -203,6 +203,7 @@ fn detect_raspi() {
 
 /// Initializes the logging system.
 pub fn init_logs() {
+    //wlroots::utils::init_logging(wlroots::utils::L_DEBUG, None);
```

Comment:
Why is this commented out?

Reply:
Do you want it on by default or toggle?

Reply:
Let's put it on by default for now. We can redecide later close to release, though I can't think of a scenario where it makes sense to disable it.